src/components/Button.tsx

```diff
@@ -37,7 +37,7 @@ const buttonStyles = css<StyledButtonProps>`
     (primary && '#FFFFFF') ||
     'black'};
   font-size: ${({ big = false }) => (big ? '1.25rem' : undefined)};
-  touch-action: manipulation;
+  touch-action: none;
```

Comment:
Have you tried removing this line? Is this line necessary?

Reply:
same either way, I can remove it.